### PR TITLE
proof of concept

### PR DESCRIPTION
**Pull request by schultzie**
Here is a proof of concept of a solution for the problem. This could be simplified by combining the code into functions but here is a long drawn out solution at least.

### CODE REVIEW

File: python_int/title_encoder.py [hunk 1]
```diff
@@ -1,9 +1,49 @@
-with open("python_core/books.txt", "r") as file:
+with open("/mnt/c/Users/wahrh/Documents/code/sololearn_snek/python_core/books.txt", "r") as file:
```
Comment (pythoninthegrass) [inline python_int/title_encoder.py:1]: Adding this:
```from pathlib import Path

# Build paths inside the project like this: base_dir / 'subdir'
base_dir = Path(__file__).resolve().parent.parent
cwd = base_dir / 'python_core'
fn = f"{cwd}/books.txt"

with open(fn, "r") as file:
    ...
```

File: python_int/title_encoder.py [hunk 2]
```diff
@@ -1,9 +1,49 @@
...
     raw = file.readlines()
     raw_lst = [line.strip() for line in raw]
-    print(raw_lst)
+    new_lst = []
```
Comment (pythoninthegrass) [inline python_int/title_encoder.py:4]: Kludgey. Gonna see if I can [golf](https://codegolf.stackexchange.com/questions/15279/tips-for-golfing-in-bash) this in main

File: python_int/title_encoder.py [hunk 3]
```diff
@@ -1,9 +1,49 @@
...
+
+    for item in raw_lst:
+        if " " not in item:
+            little_titles.append(item[0])
+
+
+    print(str(little_titles[0])) 
+    print(str(little_titles[1]))
+    print(eat_the_book)
+    print(eat_another_book)
```
Comment (pythoninthegrass) [inline python_int/title_encoder.py:42]: Clever. I'll see if it accepts more books

**Review by pythoninthegrass — APPROVED** [review]

File: python_int/title_encoder.py (hunk 3 above)
Comment (schultzie) [reply python_int/title_encoder.py:42]: We probably want to move the eat books work inside of a function that we can reuse for all of the books if more are to be added.

**Pull request merged**